fitbit-react/src/Reducers/wishlist-reducer.js

```diff
@@ -0,0 +1,9 @@
+const wishListReducer = (wishlist, action) => {
+  switch (action.type) {
+    case "SET_WISHLIST":
+      return [...action.payload.value];
+    default:
+      return wishlist;
+  }
+};
+export { wishListReducer };
```

Comment:
Suggestion you could have also managed this with only useState as it doesn't involve big complex logic

Reply:
Ya i can but for consistency i preffered this